awsagentprovider/build.gradle.kts

```diff
@@ -34,6 +34,8 @@ dependencies {
   implementation("io.opentelemetry.contrib:opentelemetry-aws-xray")
   // AWS Resource Detectors
   implementation("io.opentelemetry.contrib:opentelemetry-aws-resources")
+  // Json file reader
+  implementation(group = "com.fasterxml.jackson.dataformat", name = "jackson-dataformat-yaml", version = "2.15.1")
```

Comment:
Updated the dependency. As for storing in file vs in class, I'm not sure that matters much since now it it will read the file only once when the class is first initialized/called and after that, the list is stored in memory anyway.

Reply:
Ok. I'm fine with reading the keywords from the file for now.